src/mmrelay/log_utils.py

```
@@ -208,28 +212,33 @@ def _resolve_log_file(args: argparse.Namespace | None) -> str:
-        return os.getcwd()
-    return _get_log_dir()
+        str: The path to the logs directory.
+    
+    Raises:
+        TypeError: If the resolved `logs_dir` value is not a string.
+    """
+    from mmrelay.paths import resolve_all_paths
+
+    result = resolve_all_paths()["logs_dir"]
+    if not isinstance(result, str):
+        raise TypeError("logs_dir must be a string")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)